ddtrace/tracer/tracer_test.go

```diff
@@ -1932,9 +1932,20 @@ func (w *testTraceWriter) Flushed() []*span {
 
 func TestFlush(t *testing.T) {
 	tr, _, _, stop := startTestTracer(t)
+	defer stop()
+
 	tw := newTestTraceWriter()
 	tr.traceWriter = tw
-	defer stop()
+
+	ts := &testStatsdClient{}
+	tr.statsd = ts
+
+	transport := newDummyTransport()
+	c := newConcentrator(&config{transport: transport}, 500000)
```

Comment:
```suggestion
	c := newConcentrator(&config{transport: transport}, defaultStatsBucketSize)
```
Should we just use the default as we do in many of the tests in stats_test.go?